.gitmodules

```diff
@@ -372,3 +372,9 @@
 [submodule "contrib/chdig"]
 	path = contrib/chdig
 	url = https://github.com/azat/chdig.git
+[submodule "contrib/nlohmann-json"]
```

Comment:
That will be the 3rd JSON parser in ClickHouse after RapidJSON and SIMDJson. Is it really needed?

Reply:
Nah, not a tempting option.